packages/@tinacms/toolkit/src/components/media/media-manager.tsx

```diff
@@ -341,7 +345,19 @@ export function MediaPicker({
     }
   }
 
-  useEffect(disableScrollBody, [])
+  function filterMedia(item: Media) {
+    if (!filter) return true
+    const mime = getMime(item.filename)
+    if (filter === 'video') {
+      return mime === 'video'
+    } else if (filter === 'image') {
+      return mime === 'image'
+    } else {
+      return false
+    }
```

Comment:
```suggestion
return mime === filter
```
^ Could we simplify this down to that conditional?